shadowrunners/margrond.md

```diff
@@ -0,0 +1,136 @@
+
+| ESS | Edge | Initiative | Physical Limit | Mental Limit | Social Limit |
+| --- | --- | --- | --- | --- | --- |
+| 0.1 | 1 | 1d6 + 7 | 8 | 3 | 2 |
+
+## Condition Monitor
+__Physical:__ 11
+
+__Stun:__ 10
+
+__Overflow:__ 6
+
+##Augmentations
+| Augmentation | Rating/Attributes | Notes | Essence |
+| --- | --- | --- | --- |
+| Bone Lacing | BOD +2, Armor +2, Unarmed Damage (STR+2)P | - | 1.0 |
+| Cyberwear Commlink (Transys Avalon) | 6 | - | 0.2 |
+| Cybereyes | 3 | Flare Compensation, Low-light Vision, Vision Magnification, Vision Enhancement (1), Smartlink [Remaining Capacity 3] | 1.0 |  
+| Cyberears | 3 | Audio Enhancement (1), Damper [Remaining Capacity 10] | 0.7 |
+| Synthetic Arm (Left) | AGI 7, STR 8, Armor 2 | AGI Enhancement +2, STR Enhancement +2, Armor Enhancement +2, (+2 AGI, +3 STR Customization), -8 Concealable [Remaining Capacity 2] | 1.0 |
+| Synthetic Arm (Right) | AGI 7, STR 8, Armor 2 | AGI Enhancement +2, STR Enhancement +2, Armor Enhancement +2, (+2 AGI, +3 STR Customization), -8 Concealable [Remaining Capacity 2] | 1.0 |
+| Synthetic Leg (Left) | - | Cyber Holster, Large Smuggling Compartment, -8 Concealable | 1.0 |
+
+##Melee Weapons
+| Weapon | Accuracy | Reach | Damage | AP |
+| --- | --- | --- | --- | --- |
+| Forearm Snap-blades | 4 | - (1) | (STR+2)P | -2 |
+| Katana | 7 | 1 (2) | (STR+3)P | -1 |
+| Combat Axe | 4 | 2 (3) | (STR+5)P | -4 |
+
+##Ranged Weapons
+
+| Weapon | Damage | Accuracy | AP | Mode | RC | Ammo | Notes |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| FN HAR | 10P | 5(7) | -2 | SA / BF / FA | 2 | 35 (c) | Smartlink [integral], Periscope [Top], Gas-vent 2 [Barrel], Gyro Mount [Underbarrel]
+| Ares Predator V | 8P | 5(7) | -1 | SA | - | 15 (c) | Smartlink [integral], Periscope [Top], Gas-vent 3 [Barrel]|
+| Ares Crusader II | 7P | 5(7) | - | SA / BF | 2 | 40 (c) | Smartlink [integral], Periscope [Top], Gas-vent 2 [Barrel] | 
+| SCK Model 100 | 8P | 5(7) | - | SA / BF | (1) | 30 (c) | Smartlink [integral], Periscope [Top], Gas-vent 1 [Barrel] |
+
+##Armor
+| Armor | Rating | Notes |
+| --- | --- | --- |
+| Armor Jacket | 12 | Chemical Protection (3), Fire Resistance (3), Insulation (3), Nonconductivity (3) |
+
+##Qualities
+| Quality | Notes |
+| --- | --- |
+| Uncouth (N) | Social Tests vs. acting improperly at -2; cost for learning/improving Social Skills is double; cannot learn Social Skill groups; treated as “unaware” in Social Skills with < 1 Rating; must sometimes take tests on “for granted” social actions |
+| Uneducated (N) | Treated as “unaware” in Technical, Academic Knowledge, and Professional Knowledge skills not possessed; must sometimes take tests on “for granted” knowledge-related tasks; Karma to advance skills in above categories is doubled; possible not to be able to learn those skills | 
+| Ambidextrous (P) | No -2 for Off Hand | 
+| Toughness (P) | +1 to BOD on Damage Resistance tests | 
+| Magic Resistance 2 (P) | +2 to Spell Resistance tests |
```

Comment:
I do have the DocWagon contract, which should provide some on the spot medical attention for me. That's a good point about the magical healing, but I figured I REALLY need some better magic/tech defenses, and this was a fairly cheap version of that.

You'll just have to keep the mages off my back. :smile: